### PR TITLE
Adding option to fetch the box directly from Vagrant Cloud when runni…

### PR DESCRIPTION
**Pull request by teonivalois**
Adding possibility to use 'vagrant up 2016-box' and fetch the box from Vagrant Cloud.

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -17,6 +17,11 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     cfg.vm.provision "shell", path: "scripts/create-machine.ps1", args: "-machineHome #{home} -machineName 2016"
   end
 
+  config.vm.define "2016-box" do |cfg|
+    cfg.vm.box     = "StefanScherer/windows_2016_docker"
+    cfg.vm.provision "shell", path: "scripts/create-machine.ps1", args: "-machineHome #{home} -machineName 2016"
```
Comment (StefanScherer) [inline Vagrantfile]: Can you change the args to `-machineName 2016-box` so the vagrant vm name matches the docker machine name?

**Review by StefanScherer — COMMENTED** [review]
Thanks. This is a very nice trick. I always thought how to provide a way to use the prebuilt vagrant box or a locally built box.

One minor thing in the Vagrantfile to be ready for merge.

**Comment by teonivalois** [comment]
Done :-)

**Review by StefanScherer — COMMENTED** [review]
Thanks. LGTM

**Pull request merged**